src/api/app/views/webui2/webui/package/_side_links.html.haml

```diff
@@ -1,66 +1,24 @@
 -# FIXME: refactor this file
 %ul.list-unstyled
   - if failures > 0
-    %li
-      %i.fas.fa-times-circle.text-danger{ title: 'Errors' }
-      = failures
-      = link_to "error#{failures == 1 ? '' : 's'}", project_monitor_path(project: project, pkgname: package.name, succeeded: 0,
-                                                                         blocked: 0, finished: 0, signing: 0, dispatching: 0,
-                                                                         scheduled: 0, building: 0)
+    = render partial: 'show_failures', locals: { failures: failures, pkgname: package.name, project: project }
```

Comment:
[Nitpick]: Could you change `pkgname` to `package_name`, please

Reply:
if you are saying to change it in the view, doable nitpick, but I won't touch in the controller https://github.com/vpereira/open-build-service/blob/da08d94faa5c84c1ba3bc175c38c61584976cad2/src/api/app/controllers/webui/project_controller.rb#L390-L391 where the GET param name is ```pkgname```

Reply:
@vpereira, only in the view :+1: